SmartTV-AGH.txt

```diff
@@ -0,0 +1,250 @@
+||dpu.samsungelectronics.com^
+||kpu.samsungelectronics.com^
+||upu.samsungelectronics.com^
+||zpu.samsungelectronics.com^
+||xpu.samsungelectronics.com^
+||ypu.samsungelectronics.com^
+||vd.contents.prod.eu.s3.amazonaws.com^
+||log-ingestion-eu.samsungacr.com^
+||osb-ussvc.samsungqbe.com^
+||otn.samsungcloudcdn.com^
+
+! HBBTV
+||hbbtv.smartclip.net^
+||hbbtv-1.eurosport.com^
+||hbbtv-extern-fe01.sim-technik.de^
+||hbbtv-track.redbutton.de^
+||hbbtv.*.de^
+||hbbtv01p.anixe.net^
+||hbbtvapp.sonnenklar.tv^
+||stats.hbbtv.smartclip.net^
+||p-hbbtv.superrtl.de^
+@@||hbbtv.zdf.de^
+@@||hbbtv.prosieben.de^
+@@||hbbtv.redbutton.de^
+
+! Other useless connections from Smart-TV
+||ad.71i.de^
+||api.nfl.com^
+! apicache.vudu.com # needed for vudu app see issue22
+||cdn.smartclip.net^
+||cdns-content.dzcdn.net^
+||cert-test.sandbox.google.com^
+||database01p.anixe.net^
+||de.ioam.de^
+!drscdn.500px.org # blocks 500px on desktop
+||geo.opera.com^
+||googleads.g.doubleclick.net^
+! itv.ard.de # ARD media lib - HBBTV 
+||nbc-jite.nbcuni.com^
+||redbutton-adproxy-lb-prod.redbutton.de^
+||redbutton-lb-prod.redbutton.de^
+||redbutton.sim-technik.de^
+||script.ioam.de^
+||start.digitaltext.rtl.de^
+||stats-irl.sxp.smartclip.net^
+||tv-static.scdn.co^
+!tv.deezer.com # Breaks Deezer's smart-TV apps.
+||xml.opera.com^
+
+! Netflix
+! secure, api-global, and appboot break Netflix
+!secure.netflix.com                   
+!api-global.netflix.com
+!appboot.netflix.com
+||ichnaea.netflix.com^
+||customerevents.netflix.com^
+||nrdp.nccp.netflix.com^
+
+! Spotify
+||api-tv.spotify.com^
+
+! Hulu
+||api.distribution.hulu.com^
+
+! Sharp Smart TV using Opera OS (thanks to sml156)
+! time-a.timefreq.bldrdoc.gov          # probably not a good idea to block this one
+! api.accuweather.com                  # probably not a good idea to block this one
+
+! These may be needed for software/firmware updates, not sure if it's one or both but the first one tries thousands of times a day to connect.
+||api.*.hismarttv.com^
+
+! —————————————————————————————————————————————————————————————
+
+! Entries based on https://raw.githubusercontent.com/Perflyst/PiHoleBlocklist/master/AmazonFireTV.txt
+
+! Amazon Fire TV (First-party)
+||device-messaging-na.amazon.com^
+||devicemessaging.us-east-1.amazon.com^
+||fls-*.amazon.com^
+||mads-eu.amazon.com^
+||mas-sdk.amazon.com^
+||mas-ext.amazon.com^
+||aax-eu.amazon-adsystem.com^
+||msh.amazon.co.uk^
+||amazonadsi-a.akamaihd.net^
+||mobileanalytics.us-east-1.amazonaws.com^
+
+! Amazon Fire TV (Third-party)
+||config.ioam.de^
+||de.ioam.de^
+||secure-eu.imrworldwide.com^
+||logs1409.xiti.com^
+||tracksrv.zdf.de^
+||settings.crashlytics.com^
+
+! —————————————————————————————————————————————————————————————
+
+! Entries based on Game Console Adblock List (https://raw.githubusercontent.com/DandelionSprout/adfilt/master/GameConsoleAdblockList.txt), which the owner has donated to this project and relicenced into MIT to fit with the rest of this list
```

Comment:
My list is, to the best of my knowledge, the only console list known to exist (except for one hosts proxy file to connect to RiiConnect24's third-party online server), so you're more than welcome to submit any PS4 entries to my list if you want to, instead of having me combine these two lists and then deleting my console-only list.